backend/tests/integration/records/serializers/json/test_jobs.py

```diff
@@ -252,3 +252,29 @@ def test_jobs_detail_json_link_alias_format(inspire_app):
         response = client.get(f"/jobs/{record['control_number']}?format=json")
     assert response.status_code == expected_status_code
     assert response.content_type == expected_content_type
+
+
+def test_jobs_detail_serialize_experiment_with_referenced_record(inspire_app):
+    expected_status_code = 200
+    expected_accelerator_experiments = [
+        {
+            "name": "LIGO",
+            "record": {"$ref": "http://labs.inspirehep.net/api/experiments/1110623"},
+        }
+    ]
+    headers = {"Accept": "application/vnd+inspire.record.ui+json"}
+    record = create_record("job")
+    record["accelerator_experiments"] = [
```

Comment:
I don't get how does this even work, but can you put this data while creating the record `create_record("job", data=...)`

Reply:
Sure, but we have a lot of tests where we update the record created by ```create_record()``` .